src/actions/guides/database/intro_to_avram_and_orms.cr

```diff
@@ -9,91 +9,125 @@ class Guides::Database::IntroToAvramAndORMs < GuideAction
+    pattern breaking out where business logic is stored vs. where database queries are handled. This pattern
+    helps to keep code organized by it's purpose within your application. This includes `Models`, `Queries`,
+    `Operations`, `Migrations`, and `Boxes`.
 
     ## Models
 
-    Avram models are classes that map each database table to a specific object. The naming convention for each model is to be the singular word as where the table name is the plural form. (e.g. `User` model, and `users` table.).
+    Avram's models use classes to map each database table to a specific object. The naming convention for
+    each model is to be the singular word as where the table name is the plural form.
+    (e.g. `User` model, and `users` table.).
 
-    A model inherits from `Avram::Model`; however, Lucky will generate an abstract class called `BaseModel` for you to inherit from. This abstraction layer allows you to apply special callbacks, attributes, or methods to all of your models.
+    A model inherits from `Avram::Model`; however, Lucky will generate an abstract class called `BaseModel`
+    for you to inherit from. This abstraction layer allows you to apply special callbacks, attributes,
+    or methods to all of your models.
 
     [Learn more about models](#{Guides::Database::Models.path})
 
     ## Queries
 
-    Avram queries are classes that encompass querying the database. Once you have data in your database, you'll need a way to pull that data back out, and a Query class is that way!
-    The naming convention for these objects is to use the referencing model's name followed by `Query`. (e.g. `User` model `UserQuery` query.).
+    Avram queries are classes that encompass querying the database. Once you have data in your database,
+    you'll need a way to pull that data back out, and a Query class is that way! The naming convention
+    for these objects is to use the referencing model's name followed by `Query`.
+    (e.g. `User` model `UserQuery` query).
 
-    Query objects inherit from a subclass of the model called `BaseQuery`. If you create a `User` model, Lucky will create a `User::BaseQuery` class for you, and your `UserQuery` object will inherit from that. `UserQuery < User::BaseQuery`.
+    Query objects inherit from a subclass of the model called `BaseQuery`. If you create a `User` model,
+    Lucky will create a `User::BaseQuery` class for you, and your `UserQuery` object will inherit from that.
+    (e.g. `UserQuery < User::BaseQuery`).
 
     [Learn more about queries](#{Guides::Database::Querying.path})
 
-    ## Forms
+    ## Operations
 
-    Avram forms are classes used for creating and updating records in the database. This process is generally for interaction with users through HTML forms.
+    Avram operations are classes used for creating and updating records in the database. This process
+    is generally for interaction with users through HTML forms, but it could be used anywhere in the app
```

Comment:
I think we could remove this whole sentence.  It  is  really equally  suited to HTML forms, JSON, seeds, e tc. so this may just confuse things if we keep  the sentence